example/test.py

```diff
@@ -37,6 +37,22 @@
     'amqp://{user}:{password}@{service}/{vhost}'.format(**RABBITMQ_PARAMS))
 
 
+def bool_environ(key):
+    value = os.environ.get(key)
+    if value is None:
+        return False
+
+    value = value.lower()
+    return value == '1' or value == 'yes' or value == 'y' or value == 'true'
```

Comment:
`return value in ['1', 'yes', 'y', 'true']`? Maybe lowercasing it first to handle `Y` and `tRuE`?